resource/config.js

```diff
@@ -12,6 +12,7 @@ var ZOTERO_CONFIG = {
 	API_URL: 'https://api.zotero.org/',
 	STREAMING_URL: 'wss://stream.zotero.org/',
 	RECOGNIZE_URL: 'https://recognize.zotero.org/',
+	RESOLVE_URL: 'https://resolve.zotero.org/',
```

Comment:
We can use SERVICES_URL for this, which we're already mapping to another API Gateway endpoint for OA PDF lookup.